package/src/contexts/themeContext/utils/theme.ts

```diff
@@ -909,6 +910,11 @@ export const defaultTheme: Theme = {
       metaText: {
         fontSize: 12,
       },
+      playIcon: {
+        height: 24,
+        pathFill: '#000',
```

Comment:
Use this from theme color pallete

Reply:
I might be wrong but I prefer having the default styles in the component itself. Though we can keep this here but ideally keeping the default styles in the component makes more sense.